src/components/SecondaryPanes/Frames/tests/Group.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { shallow } from "enzyme";
+import GroupComponent from "../Group.js";
+const Group = React.createFactory(GroupComponent);
+
+import FrameMenu from "../FrameMenu";
+jest.mock("../FrameMenu", () => jest.fn());
+
+function render(overrides = {}) {
+  const defaultProps = {
+    group: [{ displayName: "foo" }],
+    selectedFrame: {},
+    selectFrame: jest.fn(),
+    copyStackTrace: jest.fn()
+  };
+
+  const props = Object.assign({}, defaultProps, overrides);
+  const component = shallow(new Group(props));
+  return { component, props };
+}
+
+describe("Group", () => {
+  it("displays a group", () => {
+    const { component } = render();
+    expect(component).toMatchSnapshot();
+  });
+
+  describe("mouse events", () => {
+    it("calls FrameMenu on right click", () => {
+      const { component, props } = render();
+      const mockEvent = "mockEvent";
+      component.simulate("contextmenu", mockEvent);
+
+      expect(FrameMenu).toHaveBeenCalledWith(
```

Comment:
Hmmm i will take a look into this again. the function `FrameMenu` relies on does not return any elements... but maybe we could check via enzyme's `mount`? I agree that checking if the elements are there is preferable to checking what arguments are passed in, but in a pinch i think it is ok to check what is being passed to FrameMenu.

On the other hand, this would not be a unit test if we do this, but rather an integration test. I am torn, personally